tasks/apt.yml

```diff
@@ -80,7 +79,9 @@
   until: result is success
 
 - name: Install spatialite
-  when: openwisp2_database.engine == "django.contrib.gis.db.backends.spatialite"
+  when: |
+    openwisp2_database.engine == "django.contrib.gis.db.backends.spatialite"
+    or openwisp2_database.engine == "openwisp_utils.db.backends.spatialite"
```

Comment:
what if we use something like `"spatialite" in openwisp2_database.engine`? I think it's pretty safe and shorter.